README.md

```diff
@@ -56,6 +56,18 @@ The boring generator introduces following generators:
 - Install SimpleForm: `rails generate boring:simple_form:install --css_framework=<css_framework>`
 - Install Devise: `rails generate boring:devise:install`
 
+**If you have Boring generators installed on your system, you can use it this way:**
```

Comment:
@luathn Can we move this section to installation directly? Primarily, suggesting to use CLI else addition via gem.

Reply:
@abhaynikam Ok. I'll move it to installation section.